packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -482,11 +495,10 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
       'iconProps': menuIconProps,
       'ariaLabel': splitButtonAriaLabel,
       'aria-haspopup': true,
-      'aria-expanded': this._isExpanded
+      'aria-expanded': this._isExpanded,
+      'data-is-focusable': !isPrimaryButtonDisabled
```

Comment:
Shouldn't this always not be focusable, even if the primary action is disabled? You want the interaction with a control to always be consistent.

Reply:
I'm working with the assumption that if the primary button is disabled, we don't want to focus on the whole container and want to focus on the menu button. Would that not be the case?

Reply:
No, that would not be the case. Focus should always only go on the container and the splitButton should always behave consistently for keyboard users